src/reactions/Reaction.js

```diff
@@ -26,24 +28,79 @@ const styles = StyleSheet.create({
   frameNotVoted: {
     borderColor: '#dedede', // eslint-disable-line
   },
-  countCommon: {
+  placeHolderCount: {
     marginLeft: 4,
+    color: 'transparent'
   },
   countVoted: {
     color: BRAND_COLOR,
   },
   countNotVoted: {
     color: 'gray',
   },
+  roller: {
+    position: 'absolute',
+    top: -20,
+    right: 4,
+  },
+  rollerText: {
+    paddingTop: 3,
+    paddingBottom: 3
+  }
 });
 
+const forwardAnimationConfig = {
+  fromValue: 0,
+  toValue: 1,
+  duration: 400,
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+};
+
+const backAnimationConfig = {
+  ...forwardAnimationConfig,
```

Comment:
Sometimes a bit of duplication makes things more clear.

You can either have:
```js
const animProps = {
  duration: 400,
  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99),
};

const forwardAnimationConfig = {
   ...animProps,
  fromValue: 0,
  toValue: 1,
};
```

Or just duplicate the `duration` and `easing` props.